cirq-core/cirq/transformers/target_gatesets/sqrt_iswap_gateset_test.py

```diff
@@ -152,8 +157,12 @@ def test_sqrt_iswap_gateset_eq():
     [
         cirq.SqrtIswapTargetGateset(),
         cirq.SqrtIswapTargetGateset(
-            atol=1e-6, required_sqrt_iswap_count=2, use_sqrt_iswap_inv=True
+            atol=1e-6,
+            required_sqrt_iswap_count=2,
+            use_sqrt_iswap_inv=True,
+            additional_gates=[cirq.XPowGate, cirq.YPowGate],
```

Comment:
I'll also add extra gates in SqrtIswapTargetGateset.repr and CZTargetGateset.repr.

Do repr tests such as this one cover more than what json serialization tests cover?

Reply:
The json tests check that `eval(rep_text_from_file)` is equal to `cirq.read_json(json_text=json_from_file)` whereas the repr tests check that `eval(repr(x)) == x`. 

So, it's possible that `repr(x)` for an object `x` doesn't produce a valid text output for which `eval(repr(x))` evaluates to the same object. But, for the same object, the developers can make a mistake and copy the correct text in `.repr` file which does evaluate to the expected object. 

I've seen the latter happen in the past, where the text in `x.repr` files didn't actually match the text produced by calling `repr(x)` on the object, which is bad but is not verified by the json tests.

Reply:
Ack. I can add a repr test to CZ gateset as well. In the long term it might be useful to have a test that verifies `repr(eval(x.repr)) == x.repr`. I can open an after-1.0 issue to track this